glide-core/src/client/mod.rs

```diff
@@ -192,25 +192,28 @@ pub async fn get_valkey_connection_info(
     }
 }
 
+use redis::{TlsCertificates, retrieve_tls_certificates};
+
 pub(super) fn get_connection_info(
     address: &NodeAddress,
     tls_mode: TlsMode,
     redis_connection_info: redis::RedisConnectionInfo,
-) -> redis::ConnectionInfo {
+    tls_params: Option<redis::TlsConnParams>,
+) -> RedisResult<redis::ConnectionInfo> {
```

Comment:
What happens if this fails? (Say you pass in an invalid byte stream)?

Reply:
Fixed to remove now that it's no longer needed.